base/loading.jl

```diff
@@ -2004,7 +2004,14 @@ function _require(pkg::PkgId, env=nothing)
 
         if JLOptions().use_compiled_modules == 1
             if !generating_output(#=incremental=#false)
-                if !pkg_precompile_attempted && isinteractive() && isassigned(PKG_PRECOMPILE_HOOK)
+                if !pkg_precompile_attempted && isinteractive()
+                    if !isassigned(PKG_PRECOMPILE_HOOK)
+                        # Note: Prevent load-time Pkg.precompile via `Base.PKG_PRECOMPILE_HOOK[]=Returns(nothing)`
+                        pkgid = Base.PkgId(Base.UUID("44cfe95a-1eb2-52ea-b672-e2afdf69b78f"), "Pkg")
+                        if Base.locate_package(pkgid) !== nothing # Only try load Pkg if we can find it
+                            Base.require(pkgid)
```

Comment:
```suggestion
                        pkgid = PkgId(UUID("44cfe95a-1eb2-52ea-b672-e2afdf69b78f"), "Pkg")
                        if locate_package(pkgid) !== nothing # Only try load Pkg if we can find it
                            require(pkgid)
```
Unless I'm missing something, the `Base.` qualifications should be unnecessary inside of Base

Reply:
Absolutely. I forgot where I was